### ISSUE TITLE
Disable users in did using the AAD accountEnabled property

### ISSUE THREAD

**Issue by damsleth** [Puzzlepart/did#1052]
- [X] I have searched for existing issues to avoid creating duplicates

### TLDR
If users have `accountEnabled: false` in Azure Active Directory, they should be disabled in did as well.
Synchronization should happen from the Admin-UI

### Description
The use case for this feature is previous employees, users who have confirmed hours previously but don't anymore.   
This feature has two functional requirements. When the user has their AAD accountEnabled property set to false:
* The user **should not be visible in the did UI**
* The user's hours **must still be part of exported reports**

#### Technical
Synchronization: a button under /admin/users next to _Add new_ and _Bulk import_ that reads _Sync user properties_
<img src="https://user-images.githubusercontent.com/7300548/184321819-a9a50b56-5de9-457c-be73-0524646fd96f.png" width=600>
  
#### Other considerations
Should the user still be visible under a proposed "disabled users" section of /admin/users? If so, where does that leave us in regards to GDPR? 
What happens if the user is _deleted_ from Azure AD? (out of scope for this feature, but worth considering)
